examples/03_connectivity/plot_compare_decomposition.py

```diff
@@ -129,6 +129,23 @@
     plot_stat_map(cur_img, display_mode="z", title="Comp %d" % i,
                   cut_coords=1, colorbar=False)
 
+###############################################################################
+# Estimate explained variance per component and plot using matplotlib
+#
+# The fitted object `dictlearning` can be used to calculate the score per component
+scores = dict_learning.score(func_filenames, per_component=True)
```

Comment:
I find that scores.sum() is differnt from `dict_learning.score(func_filenames, per_component=False)`
Is it expected ?

Reply:
I have no clue. Maybe not. If you think this is not expected then I will try to fix this as well.

Reply:
I see that we use either var(X) or _sum_of_square(X), which may explain the discrepancy. Is there any reason for that ?

Reply:
No strong motivation for that. Recent push uses ```np.var``` and still there exists discrepancy. Maybe discrepancy in variance is because estimated on reduced data versus masked data?

Reply:
I just checked with n_components=1, EV scores looked same with some discrepancy in the decimals with ```fit_intercept=False```.

Reply:
@bthirion  This remains still a question. What shall we do?

Reply:
I think that this is fine: Since components are not forced to be orthogonal, the  two score may indeed differ.
So, there's nothing more to do from my perspective.